protector/models.py

```diff
@@ -488,6 +489,15 @@ def save(self, *args, **kwargs):
         super(AbstractGenericGroup, self).save(*args, **kwargs)
         self._update_member_foreign_key()
 
+    def delete(self, *args, **kwargs):
+        result = super().delete(*args, **kwargs)
+        if self.Meta.delete_protector_group:
+            GenericUserToGroup.objects.filter(
+                group_id=self.pk,
+                group_content_type__id=ContentType.objects.get_for_model(self).id
```

Comment:
```suggestion
                group_content_type_id=ContentType.objects.get_for_model(self).id
```

Potentially it works a little faster